.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I'm unsure we want to commit this to the project, though if we do, we should likely add a description stating what this is useful for debugging.